### PR TITLE
[BACK-1937] Add ability to create custodial users for clinics

### PR DESCRIPTION
**Pull request by toddkazakov**
The source of truth for permissions for custodial accounts created by clinics is the clinics service. Permissions are kept in sync with hydrophone using Mongo CDC connector.

This PR allows creating custodial users without creating custodial permissions directly in hydrophone when the custodial user is created by a clinic.

### CODE REVIEW

**Review by adinhodovic — APPROVED** [review]

File: shoreline.go [hunk 1]
```diff
@@ -101,6 +102,12 @@ func main() {
 		config.User.Salt = salt
 	}
 
+	clinicServiceEnabled, found := os.LookupEnv("TIDEPOOL_CLINIC_SERVICE_ENABLED")
+	if found {
+		config.User.ClinicServiceEnabled = clinicServiceEnabled == "true"
```
[the remark this answers is not in the document]
Comment (toddkazakov) [reply shoreline.go:107]: This is a string to boolean conversion

**Review by adinhodovic — DISMISSED** [review]

**Review by tjotala — APPROVED** [review]
LGTM :shipit:

**Pull request merged**